dom0/sd-clean-all.sls

```diff
@@ -15,9 +15,9 @@ sd-cleanup-whonix-gw-15:
     - names:
```

Comment:
Is there a reason this `cmd.run` block was not moved to sd-clean-whonix.sls` ? it might make sense to centralize whonix-ws-15-specific cleanup tasks there

Reply:
Doing this.